app/src/test/java/org/simple/clinic/di/network/CompressRequestInterceptorTest.kt

```diff
@@ -0,0 +1,4 @@
+package org.simple.clinic.di.network
+
+class CompressRequestInterceptorTest {
```

Comment:
Shouldn't this be a part of the next commit?

Reply:
This file being checked in was an accident, as described [here](https://github.com/simpledotorg/simple-android/pull/2870#discussion_r677313713). Will remove it.